pkgs/tools/misc/whatsapp-chat-exporter/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+
+, withEncryptedAndroid ? false
+, withCrypt12 ? false
+, withCrypt14 ? false
+, withCrypt15 ? false
+}:
+
+let version = "0.9.1";
+in python3Packages.buildPythonApplication {
+  pname = "whatsapp-chat-exporter";
+  inherit version;
```

```suggestion
  inherit version;
  format = "setuptools";
```
It would be better to set the format explicitly.